UPGRADE.md

```diff
@@ -51,6 +51,8 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-beta2 to Unreleased]
 ### [shopsys/project-base]
+- *(optional)* [#596 Trusted proxies are now configurable in parameters.yml file](https://github.com/shopsys/shopsys/pull/596)
+    - implement the same patches as in commits, later will be used for production.
```

Comment:
Can you describe what needs to be done in more detail? Upgrade notes that just say "redo the pull request" are not very helpful imo.

It can be short and explanatory at the same time (the link to PR is still available, when a developer wants to see the details):
```suggestion
    - for easier deployment to production, make the trusted proxies in `Shopsys\Boostrap` class loaded from DIC parameter `trusted_proxies` instead of being hard-coded
```